apps/passport-client/src/zapp/ZappServer.ts

```diff
@@ -212,11 +213,44 @@ export class Identity extends BaseZappServer implements ZupassIdentity {
   }
 }
 
+class PODServer extends BaseZappServer implements ZupassPOD {
+  public constructor(
+    context: StateContextValue,
+    zapp: PODPCD,
+    clientChannel: ClientChannel
+  ) {
+    super(context, zapp, clientChannel);
+  }
+
+  @safeInput(ZupassAPISchema.shape.pod.shape.query.parameters())
+  public async query(query: unknown): Promise<string[]> {
+    let q;
+    try {
+      // @todo need a better way of defining the possibly type of the query
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      q = p.deserialize(query as any);
+      console.log("querying with ", q);
+    } catch (e) {
+      console.log(e);
+      throw e;
+    }
+    const allPCDs = this.getContext().getState().pcds.getAll();
+    const pods = allPCDs
+      .filter((pcd) => pcd.type === PODPCDTypeName)
```

Comment:
noting we'll need to figure out what the strategy is for PCDs that are POD-based but not PODPCD e.g. tickets right now